frontend/packages/ceph-storage-plugin/src/components/attached-devices-mode/lso-disk-inventory/disk-replacement-modal.tsx

```diff
@@ -106,29 +114,31 @@ const DiskReplacementAction = (props: DiskReplacementActionProps) => {
 
   return (
     <form onSubmit={handleSubmit} name="form" className="modal-content">
-      <ModalTitle>Disk Replacement</ModalTitle>
+      <ModalTitle>{t('ceph-storage-plugin~Disk Replacement')}</ModalTitle>
       <ModalBody>
-        <p>This action will start preparing the disk for replacement.</p>
+        <p>{t('ceph-storage-plugin~This action will start preparing the disk for replacement.')}</p>
         {isRebalancing && alertsMap[diskName].status !== Status.Offline && (
           <Alert
             variant="info"
             className="co-alert"
-            title="Data rebalancing is in progress"
+            title={t('ceph-storage-plugin~Data rebalancing is in progress')}
             isInline
           >
             <Link onClick={close} to={DASHBOARD_LINK}>
-              See data resiliency status
+              {t('ceph-storage-plugin~See data resiliency status')}
             </Link>
           </Alert>
         )}
         <p>
-          Are you sure you want to replace <strong>{diskName}</strong> ?
+          <Trans t={t} ns="ceph-storage-plugin">
+            Are you sure you want to replace <strong>{diskName}</strong> ?
```

Comment:
I think this needs to be 
```suggestion
            Are you sure you want to replace <strong>{{diskName}}</strong> ?
```